service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -52,6 +52,7 @@
 
 @ExtendWith(SpringExtension.class)
 @ContextConfiguration(classes = {PaymentService.class})
+@TestPropertySource(properties = {"payment.site_id=TEST_SITE_ID"})
```

Comment:
Do you think it would be useful to add a unit test profile so that we don't need to add this and other properties to the tops of files. We could then also remove the TestConfig class (can't remember name) which sets up a load of test beans and just add what is needed to the context config.

Reply:
Interesting point, but not sure if such approach is correct. Ideally unit test should be real unit tests fully isolated and  should not depends on env or bootstrap spring context (or at least should inject mocks and not real dependencies) Additionally with such approach we have limited options to unit test with different injected parameters, e.g if we have flag in config that turn on/off some logic, like feature toggle, with single test profile it would be hard to provide different variations to be test.

Reply:
Yeah, that was my sort of concern as well.